packages/webcert/src/store/utils/utilsMiddleware.ts

```diff
@@ -13,11 +13,22 @@ import {
   updateDiagnosisTypeahead,
   updateDynamicLinks,
 } from './utilsActions'
+import { updateCertificate } from '../certificate/certificateActions'
+
+const handleUpdateCertificate: Middleware<Dispatch> = ({ dispatch, getState }) => (next) => (action: AnyAction): void => {
+  next(action)
```

Comment:
Suggest that you remove handleUpdateCertificate and instead dispatch getAllDynamicLinks when the app loads in App.tsx.